include/d/d_com/d_com_inf_game/d_com_inf_game.h

```diff
@@ -166,6 +181,151 @@ class dComIfG_inf_c {
+}
+inline void dComIfGp_setItemKeyNumCount(short amount) {
+    g_dComIfG_gameInfo.getPlay().setItemKeyNumCount(amount);
+}
+inline void dComIfGp_setItemMaxLifeCount(short max) {
+    g_dComIfG_gameInfo.getPlay().setItemMaxLifeCount(max);
+}
+inline void dComIfGs_onDungeonItemMap(void) {
+    g_dComIfG_gameInfo.getMemory().getTempFlags().onDungeonItemMap();
+}
+inline void dComIfGs_onDungeonItemCompass(void) {
+    g_dComIfG_gameInfo.getMemory().getTempFlags().onDungeonItemCompass();
+}
+inline void dComIfGs_onDungeonItemWarp(void) {
+    g_dComIfG_gameInfo.getMemory().getTempFlags().onDungeonItemWarp();
+}
+inline void dComIfGs_setItem(int slot, u8 i_no) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerItem().setItem(slot, i_no);
+}
+inline void dComIfGs_setOxygen(long amount) {
+    g_dComIfG_gameInfo.getPlay().setOxygen(amount);
+}
+inline void dComIfGs_setMaxOxygen(long max) {
+    g_dComIfG_gameInfo.getPlay().setMaxOxygen(max);
+}
+inline void dComIfGs_setOil(u16 amount) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerStatusA().setOil(amount);
+}
+inline void dComIfGs_setMaxOil(u16 max) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerStatusA().setMaxOil(max);
+}
+inline void dComIfGs_setWalletSize(u8 size) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerStatusA().setWalletSize(size);
+}
+inline void dComIfGs_setMagic(u8 amount) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerStatusA().setMagic(amount);
+}
+inline void dComIfGs_setMaxMagic(u8 max) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerStatusA().setMaxMagic(max);
+}
+inline void dComIfGs_setRupee(u16 amount) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerStatusA().setRupee(amount);
+}
+inline void dComIfGs_setLife(u16 amount) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerStatusA().setLife(amount);
+}
+inline void dComIfGs_setMaxLife(u8 max) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerStatusA().setMaxLife(max);
+}
+inline void dComIfGs_onDungeonItemBossKey(void) {
+    g_dComIfG_gameInfo.getMemory().getTempFlags().onDungeonItemBossKey();
+}
+inline void dComIfGs_setCollectSword(u8 sword_id) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerCollect().setCollect(SWORD_BITFIELD, sword_id);
+}
+inline void dComIfGs_setCollectShield(u8 shield_id) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerCollect().setCollect(SHIELD_BITFIELD, shield_id);
+}
+inline void dComIfGs_setCollectClothes(u8 clothes_id) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerCollect().setCollect(CLOTHING_BITFIELD, clothes_id);
+}
+inline void dComIfGs_setRodTypeLevelUp(void) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerItem().setRodTypeLevelUp();
+}
+inline void dComIfGs_setArrowNum(u8 amount) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerItemRecord().setArrowNum(amount);
+}
+inline void dComIfGs_setArrowMax(u8 max) {
+    g_dComIfG_gameInfo.getSaveFile().getPlayerItemMax().setArrowNum(max);
+}
+inline u8 dComIfGs_getPachinkoMax(void) {
```

Comment:
Have you tried compiling these without the explicit `inline`? Unless its necessary, it's cleaner without them.
```suggestion 
u8 dComIfGs_getPachinkoMax(void) {
```

Reply:
yea the way they are like this, compiling without inline makes it not match